.github/workflows/update_dependencies.yml

```diff
@@ -80,4 +80,5 @@ jobs:
           --base "$GITHUB_REF" \
           --title "$PR_TITLE" \
           --body-file "$TMP_PR_BODY_PATH" \
-          --label "$PR_LABELS"
+          --label "$PR_LABELS" \
+          --reviewer "DataDog/agent-shared-components"
```

Comment:
We can probably update this to the new team name

Reply:
Yep agreed 👍